### PR TITLE
refectory: fix misspelling

### PR DESCRIPTION
**Pull request by yj-yoo**
### What this PR does
fix misspelling

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [x] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

**Comment by aburdenthehand** [comment]
Thanks for the PR. These changes look good to me. 
Can you please sign the commit?

File: pkg/util/util.go [hunk 1]
```diff
@@ -196,7 +196,7 @@ func ResourceNameToEnvVar(prefix string, resourceName string) string {
 	return fmt.Sprintf("%s_%s", prefix, varName)
 }
 
-// Checks if kernel boot is defined in a valid way
+// HasKernelBootContainerImage Checks if kernel boot is defined in a valid way
```
Comment (0xFelix) [inline pkg/util/util.go:199]:
```suggestion
// HasKernelBootContainerImage checks if kernel boot is defined in a valid way
```

**Comment by yj-yoo** [comment]
@0xFelix Thanks☺️

**Review by 0xFelix — COMMENTED** [review]
Happy to help! :smiley: 

The changes look good to me, but I would suggest squashing the commits to a single commit. Could you do that? Thanks!

**Comment by yj-yoo** [comment]
@0xFelix 
oh. I missed something important. Thanks for letting me know. Compressed multiple commits into one.

**Review by 0xFelix — APPROVED** [review]
Thank you!

/lgtm

**Review by xpivarc — COMMENTED** [review]
/approve

**Comment by xpivarc** [comment]
/hold
@yj-yoo Please run `make generate` and commit the changes

**Pull request closed without merging**